internal/experimental/registry/reference_test.go

```diff
@@ -116,4 +116,18 @@ func TestParseReference(t *testing.T) {
 	s = "localhost:5000/x/y/z:123:x:y"
 	_, err = ParseReference(s)
 	is.Error(err, "ref contains too many colons (4)")
+
+	s = "myrepo/mychart:1222222"
+	ref, err = ParseReference(s)
+	is.NoError(err)
+	is.Equal("myrepo/mychart", ref.Repo)
+	is.Equal("1222222", ref.Tag)
+	is.Equal("myrepo/mychart:1222222", ref.FullName())
+
+	s = "myrepo/mychart:5000:1222222"
```

Comment:
why should this pass? This should be considered an error case, as this is defining the tag twice, and colons are not an accepted field for `docker tag`. `myrepo:5000/mychart:1222222` should be fine though.

```
><> docker build -t myrepo/mychart:5000:1222222 .
invalid argument "myrepo/mychart:5000:1222222" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
```

Reply:
I overlooked this. I agree, the test should be for `myrepo:5000/mychart:1222222`